site/sfguides/src/triggering_real_time_notifications_using_dynamic_tables_and_census_live_syncs/triggering_real_time_notifications_using_dynamic_tables_and_census_live_syncs.md

```diff
@@ -0,0 +1,204 @@
+author: Census
+id: triggering_real_time_notifications_using_dynamic_tables_and_census_live_syncs
+summary: In this guide, you will learn how to set up a live sync between Census and Snowflake to move data to your activation platforms in real time.
+categories: partner-integrations,marketing
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Census, Real-Time, Dynamic Tables, Reverse ETL
+
+# A Guide to Triggering Real-Time Notifications Using Dynamic Tables and Census Live Syncs
+
+## Overview
```

Comment:
@dougvalenta @sfc-gh-lambrosetti Add link to Snowflake account signup or login page in under Prereqs or What You'll Need section.

Reply:
Added